mirdata/utils.py

```diff
@@ -138,6 +138,8 @@ def validator(dataset_index, data_home, dataset_path, silence=False):
 
 KeyData = namedtuple('KeyData', ['start_times', 'end_times', 'keys'])
 
+ActivityData = namedtuple('ActivityData', ['start_times', 'end_times', 'activity'])
```

Comment:
What is activity? like an instrument is active? How about we call it `EventData`

Reply:
Moved to `EventData` internally, kept `activity` in the attribute name to relate with the title of the publication